test/parallel/test-process-features.js

```diff
@@ -15,6 +15,7 @@ assert.deepStrictEqual(keys, new Set([
   'tls_ocsp',
   'tls',
   'cached_builtins',
+  'typescript',
```

Comment:
```js
if (key === 'typescript') {
	assert.ok(['boolean', 'string'].includes(typeof process.features[key]))
	continue
}
```

could be a solution here but I agree it's not necessarily pretty. A better way would be to use a `Map` over a set with key being the feature and the value an array of accepted types